portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsBookmarks.testcase

```diff
@@ -72,4 +72,100 @@ definition {
 		}
 	}
 
+	@description = "Assert that we can resolve bookmark entry conflicts by discarding changes."
+	@priority = 4
+	test ResolveConflictingPublicationsByDiscarding {
+		property custom.properties = "feature.flag.LPS-171364=true";
```

Comment:
Allen, we don't need to add the property since there are no steps to move changes, can you remove it?

Reply:
Oops, that was leftover from before we changed the test steps; thanks for pointing that out!